t/01-hash.t

```diff
@@ -186,12 +194,14 @@ is_deeply(
             a => "A",
             b => "B",
             c => "C"
-        )
+        ),
     },
     {B => 2},
     "slice_grep_map on values",
 );
 
+is_deeply({slice_grep_map { $_{$_} && $_{$_} > 1 } \%h}, {b => 2}, "slice_grep_map use slice_grep",);
```

Comment:
test damit in folgender Zeile auch die rechte Seite durchlaufen wird.
https://github.com/perl5-utils/Hash-MoreUtils/blob/e724623873c846296e431d4de07d8f4c30caaf38/lib/Hash/MoreUtils.pm#L166